src/unix/udp.c

```diff
@@ -704,7 +704,15 @@ int uv__udp_disconnect(uv_udp_t* handle) {
 
     do {
       errno = 0;
+#ifdef __PASE__
+      // On IBMi a connectionless transport socket can be disconnected by
+      // either setting the addr parameter to NULL or setting the
+      // addr_length parameter to zero, and issuing another connect().
+      // https://www.ibm.com/docs/en/i/7.2?topic=ssw_ibm_i_72/apis/connec.htm
```

Comment:
```suggestion
      // https://www.ibm.com/docs/en/i/7.4?topic=ssw_ibm_i_74/apis/connec.htm
```

I'd recommend using the most recent version of the doc, since older release tend to get archived at some point after they go out of support (which 7.2 already is) and the links stop working.